cnxepub/models.py

```diff
@@ -128,12 +128,12 @@ def model_to_tree(model, title=None, lucent_id=TRANSLUCENT_BINDER_ID):
         {'id': <id>|'subcol', 'title': <title>, 'contents': [<tree>, ...]}
 
     """
-    if type(model) is TranslucentBinder:
+    id = model.ident_hash
+    if id is None and isinstance(model, TranslucentBinder):
```

Comment:
The cnx-epub library does not enforce identifiers. It should be possible to build unidentified content object structures. The library itself isn't concerned about identifiers or what the identifiers look like. This is why things like `cnx-archive-uri` exist. Technically, it should be possible to incorporate another provider's identifier scheme into this library.

Unfortunately I don't feel like everyone is sharing this vision of a library that is identifier and persistence layer agnostic. So I may decide to roll over on the subject. 

I'm feeling like the vision is already quite broken. And it will be increasingly broken with the incorporation of the external identifier scheme (i.e. `shortId`).